amida-tech/greyscale
# Feature/inba 329 jf upgrade pg lib

#### What does this PR do?
This PR updates the `pg` node package to 7.4.3 and prepares us somewhat for rewriting Greyscale to use Sequelize, and probably do away with Thunkify.

@au2, the issue I was having is due to version 7.4.3 also returning the `SET` command of the namespace as the first of two fields of any query. Before we were getting only an object of the real query (ie, `DELETE`, `SELECT`). These calls are made twice on lines, like, 139 and 221 of `util.js`. If you notice any magic to ignore that, let me know.

#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-329
https://jira.amida-tech.com/browse/INBA-907

#### How should this be manually tested?
Assuming you still have `su@mail.net` in your auth db and with Amida-Auth-Microservice running, blow away `indaba` and `indabatestuser` and recreate the DBs. Run `node seed.js`. A test may fail due to date comparison (need to investigate), but ensure that you have a populated DB otherwise. Then log in and do a full regression test of all features. 

If you don't have `su@mail.net`, you may need to create him or change the super user to an admin type in your auth microservice. You can change it under `backend\config\config.js` under `testEntities.superAdmin`.

#### Background/Context

#### Screenshots (if appropriate):
